File: vs/ttn/java ttn/anagram.java
import java.util.Scanner;

public class anagram {
    public static void main(String[] args) {
        String a;
        String b;
        // char i;
        int c1=0,c2=0;
        boolean flag=true;
        Scanner sc=new Scanner(System.in);
        a=sc.nextLine();
        b=sc.nextLine();
        for(char i='a';i<='z';i++)
        {
            c1=0;
            c2=0;
            for(int j=0;j<=a.length()-1;j++)
            {
                if(a.charAt(j)==i) c1++;
            }
            for(int k=0;k<=b.length()-1;k++)
            {
                if(b.charAt(k)==i) c2++;
            }
            if(c1!=c2) 
    {
        flag=false;
        System.out.println("no");
        break;
    }
        }
        if(flag)System.out.println("yes");
    }
}

File: vs/ttn/cpp ttn/anagram.cpp
//anagram
#include<iostream>
using namespace std;
int main(){
char a[50],b[50];
int c1=0,c2=0,j=0;
gets(a);
gets(b);
bool flag = true;


for(char j='a';j<='z';j++){
    c1=0;
    c2=0;
    
    for(int i=0;a[i]!='\0';i++){
        if(a[i]==j) c1++;
        
    }

    for(int k=0;b[k]!='\0';k++){
        if(b[k]==j) c2++;
        
    }

    if(c1!=c2) 
    {
        flag=false;
        cout<<"no";
        break;
    }
    
}

if(flag) cout<<"yes";







}